app/routes/collections.photographs.$photograph.tsx

```diff
@@ -38,11 +40,24 @@ const PhotographDetail = () => {
 
       <div>
         <h1 className="text-lg text-black/85">{photograph.name}</h1>
+
+        {photograph.full_url && (
+          <a
+            href={photograph.full_url.replace("/square/", "/full/")}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="text-sm text-blue-600 underline hover:text-blue-800 block mt-1"
+          >
+            View Full Image
```

Comment:
The only additional change I would like to see is some white space between the text and the icon. It's not as intuitive as one would think in JSX. In some places I've used 

~~~jsx
{" "}
~~~

to achieve the space. However, I just looked into what that actually just does. I still have very little understanding. It's probably best to just add a margin to the left of the icon - maybe try `ms-2` and see how it looks. That's another thing we can clean up whenever we come across it.